cargo-dist/src/tasks.rs

```diff
@@ -736,6 +740,18 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
             DirtyMode::AllowList(allow_dirty.clone().unwrap_or(vec![]))
         };
 
+        let mut system_dependencies = workspace_metadata
+            .system_dependencies
+            .clone()
+            .unwrap_or_default();
+
+        // Include all package-local dependencies from everything we're building
+        for package in &package_metadata {
+            if let Some(package_deps) = &package.system_dependencies {
+                system_dependencies.append(&mut package_deps.clone());
+            }
+        }
```

Comment:
We're too early in the code to make this determination -- we only have that info around here, when we know `announcing.rust_releases`, because we """need""" to be factoring in the whether they're releasing `my-app-v1.0.0` (just one app in the workspace) or `v1.0.0` (~everything).

https://github.com/axodotdev/cargo-dist/blob/815ca1096e382c3c31f074d921f4d67844c09906/cargo-dist/src/tasks.rs#L2235